Tests/SentryTests/Swift/Core/Tools/LoadValidatorTests.swift

```diff
@@ -0,0 +1,197 @@
+import ObjectiveC
+@_spi(Private) @testable import Sentry
+@_spi(Private) import SentryTestUtils
+import XCTest
+
+class LoadValidatorTests: XCTestCase {
+    
+    // MARK: - Test Properties
+    
+    private var testOutput: TestLogOutput!
+    private var testObjCRuntimeWrapper: SentryTestObjCRuntimeWrapper!
+    private var dispatchQueueWrapper: TestSentryDispatchQueueWrapper!
+    private var defaultImageAddress: UInt64 = 0x1000
+    private var defaultImageSize: UInt64 = 0x20
+    
+    // MARK: - Setup and Teardown
+    
+    override func setUp() {
+        super.setUp()
+        testOutput = TestLogOutput()
+        SentrySDKLog.setLogOutput(testOutput)
+        SentrySDKLog.configureLog(true, diagnosticLevel: .debug)
+        
+        testObjCRuntimeWrapper = SentryTestObjCRuntimeWrapper()
+        
+        dispatchQueueWrapper = TestSentryDispatchQueueWrapper()
+        dispatchQueueWrapper.dispatchAfterExecutesBlock = true
+    }
+    
+    override func tearDown() {
+        clearTestState()
+        super.tearDown()
+    }
+    
+    // MARK: - Test Cases
+    
+    func testValidateSDKPresenceIn_SystemLibraryPath_DoesNotValidate() {
+        // Arrange
+        let imageName = "/usr/lib/system.dylib"
+        var getClassListCalled = false
+        testObjCRuntimeWrapper.beforeGetClassList = {
+            getClassListCalled = true
+        }
+        
+        // Act
+        let validationResult = LoadValidator.checkForDuplicatedSDKInSync(imageName,
+                                                                         defaultImageAddress,
+                                                                         defaultImageSize,
+                                                                         objcRuntimeWrapper: testObjCRuntimeWrapper,
+                                                                         dispatchQueueWrapper: dispatchQueueWrapper)
+        
+        // Assert
+        XCTAssertFalse(validationResult, "Validation should skip for system libraries")
+        XCTAssertFalse(getClassListCalled, "ObjectiveC Wrapper shouldd not be called for a system library")
```

Comment:
`l`: The same typo exists multiple times.
```suggestion
        XCTAssertFalse(getClassListCalled, "ObjectiveC Wrapper should not be called for a system library")
```